onelogin_rules/onelogin_user_account_locked.py

```diff
@@ -0,0 +1,9 @@
+def rule(event):
+
+    # check for a user locked event
+    # event 531 and 553 are user lock events via api
+    return event.get('event_type_id') in [531, 553]
```

Comment:
I scoped this down pretty heavily. Is it appropriate to do so? For example, separate rules for user suspended vs user locked

Reply:
I think so. Currently, we cannot dynamically set the severity of an alert. So if two rules are similar but have slightly different meanings based on for example what event code occurred, it is reasonable to make them two separate rules so that their severity can be set appropriately.